components/match2/commons/match_group_input_util.lua

```diff
@@ -1283,9 +1284,11 @@ function MatchGroupInputUtil.standardProcessMaps(match, opponents, Parser)
 			map.winner = MatchGroupInputUtil.getWinner(map.status, winnerInput, map.opponents)
 		end
 
+		dateProps.date = nil
 		map.extradata = Table.merge(
 			{displayname = map.mapDisplayName},
-			Parser.getExtraData and Parser.getExtraData(match, map, opponents) or nil
+			Parser.getExtraData and Parser.getExtraData(match, map, opponents) or nil,
+			dateProps
```

Comment:
If we're gonna store the full dateProps, should remove the dateexact from getExtraData that most FFA wikis has.